libV2/constants.js

```diff
@@ -55,4 +55,13 @@ module.exports = {
         outgoingBytes: 'outB',
     },
     warp10ValueType: ':utapi:event',
+    truthy: [
+        'true',
+        'on',
+        'yes',
+        'y',
+        't',
+        'enabled',
+        'enable',
+    ],
```

Comment:
a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) can be used rather than an array.

Reply:
What upsides does a set have here over a static array?

Reply:
Sets are generally better for pure lookups as they are hashed so O(1) lookups.

Reply:
Preferably if javascript would freeze them that would be cool but there is no immutable sets in the standard packages as far as I know. Python has frozen sets https://docs.python.org/3/library/stdtypes.html#frozenset which are even better.

Reply:
makes sense